### ISSUE TITLE
C64 CIA window problem.

### ISSUE THREAD

**Issue by arsoft** [slajerek/RetroDebugger#3]
Hi Marcin,
There is a font scaling problem in the CIA status window. In general, when the font size is changed, the CIA 2 fields are rendered in their original position and obscure some of the CIA 1 state information.

I think the problem is somewhere here: 
```C++
void CViewC64StateCIA::Render()
{
//	if (debugInterface->GetSettingIsWarpSpeed() == true)
//		return;
	
	float px = posX;
	if (renderCIA1)
	{
		this->RenderStateCIA(px, posY, posZ, fontBytes, fontSize, 1);
		px += fontSize + 190;
	}
	
	if (renderCIA2)
	{
		this->RenderStateCIA(px, posY, posZ, fontBytes, fontSize, 2);
	}
}
```
or at other place when is used magic 190 number
![C64_CIA_Window](https://user-images.githubusercontent.com/2960610/147477721-24e74e9d-aef1-48a4-800c-c20ee953e4ec.png)

**Comment by slajerek** [comment]
Thanks for reporting. Indeed that was a leftover from old view version when everything had fixed sizes. I've fixed this in commit #570f22fc4eab70c3156119727f8630adc50b7d2c

**Comment by arsoft** [comment]
Well now it's looks much better. Thanks.
